hp/metadata_collector.py

```diff
@@ -96,12 +96,20 @@ def get_text(text_path):
             # vyhledá počet slov
             wordcount = re.search(r'<dd class="words">((?:\d*,?)+)</dd>', part).groups(1)[0].replace(",", "")
             subresult.append(wordcount)
+            # vyhledá datum publikování
+            published= re.search (r'<dd class="published">(\d\d\d\d-\d\d-\d\d)</dd>', part).group(1)[0]
```

Comment:
are you sure you need `[]` in the end? I think it will return first digit of year only